run_desktop_tests.sh

```diff
@@ -23,6 +23,9 @@ cd ../..
 cd libraries/crypto
 cargo fmt --all -- --check
 cd ../..
+cd libraries/rng256
+cargo fmt --all -- --check
+cd ../..
```

Comment:
What about the github workflows?

Reply:
I added one that is almost a copy of the crypto workflow!